### ISSUE TITLE
Steam Truck Storage Interaction

### ISSUE THREAD

**Issue by Frosty939** [StrangeLoopGames/EcoSuggestions#603]
When you try to open the storage area of the truck, it opens to the fuel page instead of storage.

It would be super if it would open the storage area of the truck when you interact with the storage area lol

**Comment by Tharalo** [comment]
Right... and you should take the first available material out of the storage when you hit it with the hammer, not the coal out of the fuel stock.
